app/scripts/actions/index.js

```diff
@@ -14,6 +15,10 @@ export const LOGIN = 'LOGIN';
 export const LOGIN_INFLIGHT = 'LOGIN_INFLIGHT';
 export const LOGIN_ERROR = 'LOGIN_ERROR';
 
+export const ADD_CMR = 'ADD_CMR';
```

Comment:
Nitpick: Change to `ADD_CMR_INSTANCE_META`

Reply:
how about ADD_INSTANCE_META_CMR?